### DISCUSSION TITLE
interpretation of advanced parameters - confirmed category

### DISCUSSION THREAD

**Discussion by kaiserso** [neherlab/covid19_scenarios#405] [General] [answered]
Hi,

I found the 'confirmed' category on advanced parameters confusing.  While the proportion of cases that are confirmed is <1 and varies significantly between settings, the tool seems to be using the probability of cases being detected as a proxy for risk of disease progression but they are not quite the same.  If you run out of tests in a country you will have zero confirmed cases but they will progress to hopsitalization and death nonetheless.  Thus I would not expect the confirmation rate to directly impact the hospitalization rate, but it does.  The defaults result in very low assumptions for hospitalization, critical care and deaths.  I thought about using this to model the asymptomatic proportion, however asymptomatic probably have different transmission probabilities, so this is not entirely appropriate, but probably better than the current usage. As it is a bit hidden away, I think this should either be set to 100% (or 70% to represent symptomatic) by default to avoid misleading results, or this should be redesigned to include an asymptomatic compartment which has a different risk of transmission and no risk of progression.  The confirmation proportion could then be used to mirror/match reported case data only and not otherwise influence the results.

A way to see why this does not appear to have reasonable defaults is that the defaults are described as based on the China CDC data but the final IFRs that are computed from these settings results in much lower IFRs than Verity et al (Lancet) published after adjusting the China CDC data.

Does this make sense?

thanks,
Peter

**Comment by noleti** [comment, marked as answer]
I agree on 'confirmed' being somewhat confusing. First off, I note that the model assumes that there are always sufficient hospital beds available, and all severe cases will go to hospital. As there is much higher hospital bed capacity than IC(M)U beds, I think this is reasonable (as people die in critical stage, not in severe). Alternatively, if you assume that the chance to progress from severe to critical is the same in and out of hospital, those simplifications should not matter. I agree that the model currently assumes that anyone infected has the same chance to transmit, regardless of symptoms (which are not modeled apart from progression to severe). I also note that severe cannot infect others any more (which is again assumed they are hospitalized in an ideal condition).

I assume the reason to have 'confirmed' is to have an explicit way to model overestimations in reported IFR due to selection bias (only severe cases are tested). I'm not a domain expert, but I think this model is more intended to estimate the infected numbers based on deaths (while you might assume it works the other way around). Extrapolation of cases are thus relying on real observations of deaths, and we need a way to capture undercounting of infected (but not severe) cases. If there are no deaths or hospitalized yet for a country and parameters are chosen to match known infected, this model likely underestimates the number of deaths (as real infectious numbers are likely much higher than known).

If you really don't like the 'confirmed', you should be able to set it to 100 and lower 'severe' appropriately (by setting it to the old confirmed*severe). The resulting numbers should not change.

I will let the domain experts comment on  IFRs by Verity et al (Lancet).

**Reply by kaiserso** [reply]
Thank you for this response. I also re-read Verity again along with the appendices and although I'm still somewhat confused by that paper and how they obtained their adjusted CFRs and IFRs, I now think the modeling of 'confirmed' is consistent with their usage/approach.  The defaults though do seem to be giving age-specific IFRs that are notably below Verity et al (and Ferguson et al, if I am not mistaken), which is surprising.

I think you have correctly identified that the model may work better when fitting deaths and hospitalized cases rather than trying to estimate from infections and cases (which is what I am doing, for Kenya, where the number of deaths to date is quite few).

Thank you for taking the time to respond!
